5.sc_Morphology_Logistic_Regression/1.train_models/models/single_class/PBMC/aggregated/DMSO_0.100_DMSO_0.025__LPS_1.000_DMSO_0.025/shuffled_baseline__CP.toml

```diff
@@ -0,0 +1,6 @@
+model_type='shuffled_baseline'
```

Comment:
What is the advantage of a toml file, and what is a toml file, curious?

Reply:
In my opinion better readability than a `.yaml` file but same functionality.